configure.in

```diff
@@ -7023,22 +7023,35 @@ fi
 
 if test "$wxUSE_WEBVIEW" = "yes"; then
     USE_WEBVIEW_WEBKIT=0
+    USE_WEBVIEW_WEBKIT2=0
     if test "$wxUSE_WEBVIEW_WEBKIT" = "yes"; then
         if test "$wxUSE_GTK" = 1; then
             webkitgtk=webkit-1.0
             if test "$WXGTK3" = 1; then
                 webkitgtk=webkitgtk-3.0
```

Comment:
Unless I'm missing something, this won't change the functionality as that variable isn't used in the webkit2 case, but if you think it makes more logical sense to move it down there, OK.